docs/multi-storage-transactions.md

```diff
@@ -49,7 +49,7 @@ scalar.db.multi_storage.namespace_mapping=user:cassandra,coordinator:mysql
 scalar.db.multi_storage.default_storage=cassandra
 ```
 
-## References
+## Further documentation
```

Comment:
```suggestion
## Further documents
```
Probably. I think Documentation sounds more like creating a document.

Reply:
I fixed it in b80bcbd9bae2806b55c39106ac41cd08ac9a59e0. I also modified the same for other parts.